go.mod

```diff
@@ -43,3 +43,6 @@ replace github.com/cosmos/iavl => github.com/cosmos/iavl v0.17.1
 
 // FIXME: https://github.com/crypto-org-chain/ethermint/tree/cronos2
 replace github.com/tharsis/ethermint => github.com/crypto-org-chain/ethermint v0.4.2-0.20211004101819-7b5449f36cc8
+
+// TODO: remove when ibc-go and ethermint upgrades cosmos-sdk
+replace github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.44.2
```

Comment:
would that replace the cosmos-sdk dependency in ibc-go and ethermint without any side effect?

Reply:
Hard to say about side-effect, but I think it's safer than using different versions at the same time.